src/ui/components/DevTools.js

```diff
@@ -44,18 +46,24 @@ function DevtoolsSplitBox({ toolboxExpanded, updateTimelineDimensions, tooltip }
   };
 
   return (
-    <SplitBox
-      style={{ width: "100vw", overflow: "hidden" }}
-      splitterSize={1}
-      initialSize={prefs.toolboxHeight}
-      minSize="20%"
-      maxSize="80%"
-      vert={false}
-      onMove={handleMove}
-      startPanel={viewer}
-      endPanel={toolbox}
-      endPanelControl={false}
-    />
+    <>
```

Comment:
why do we need this?

Reply:
Which one are you referring to?

If it's the splitbox, then it's for resizing the left section (primary toolbar, sidebar, editor) and the right section (viewport, secondary toolbar).

Reply:
the outer `<>`